AUTHORS.md

```diff
@@ -0,0 +1,49 @@
+# Credits
+
+---
+© 2024 **xDEM developers**.
+
+**xDEM** is licensed under permissive Apache 2 license (See LICENSE file).
+
+All contributors listed in this document are part of the **xDEM developers**, and their
+contributions are subject to the project's copyright under the terms of the
+[Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+
+See CONTRIBUTING.md for more details on Contributor License Agreement.
+
+This file keeps track of authors contributions.
+
+## Development Lead
+
+---
+
+- **Romain Hugonnet** [@rhugonnet](https://github.com/rhugonnet) <romain.hugonnet@gmail.com>
+- **Amaury Dehecq** [@adehecq](https://github/adehecq) <amaury.dehecq@univ-grenoble-alpes.fr>
+- **Erik Schytt Mannerfelt** [@erikmannerfelt](https://github/erikmannerfelt)
+
+## Contributors
+
+---
+
+- **Friedrich Knuth** [@friedrichknuth](https://github/friedrichknuth)
+- **Andrew Tedstone** [@atedstone](https://github/atedstone)
+- **Zhihao LIU** [@liuh886](https://github/liuh886)
+- **Diego Cusicanqui** [@cusicand](https://github/cusicand)
+- **Alessandro Gentilini** [@alessandro-gentilini](https://github/alessandro-gentilini)
+- **Ferdinand Schenck** [@fnands](https://github/fnands)
+- **Johannes Landmann** [@jlandmann](https://github/jlandmann)
+- **Valentin Schaffner** [@vschaffn](https://github/vschaffn) <valentin.schaffner@cs-soprasteria.com>
```

Comment:
Need to add @MatteaE as a contributor here as well (from the list of committers).
And also @ameliefroessl for all her work on Dask in https://github.com/GlacioHack/xdem/discussions/501 and #525, even if this has been on pause since the summer (sorry, too much to do this fall!) :wink:.

Other actors I'm tagging that can tell us if they want to be added:
@fmaussion: You helped a lot starting up the package during the first hackweek, advising on good practices for CI/doc/tests and sharing code for that (in addition to discussing about the landscape for Xarray geospatial). You have commits on GeoUtils but not here, I still feel like you should be added as contributor. What do you think?
@sgascoin: For providing a base algorithm + tips on the optimizer for a new coregistration method, if you feel it is warranted.

Reply:
Sure, I'm happy to be in the friends list!

Reply:
@fmaussion Reminder on this